tests/performance/range_search/create_docs.cpp

```diff
@@ -28,7 +28,7 @@ IntVector make_range_values(int num_docs, int values_in_range) {
         int hits = ((size_t)num_docs * (size_t)hits_ratio) / 1000;
         if (hits >= values_in_range) {
             int hits_per_value = hits / values_in_range;
-            int value = hits_ratio * 100000 + values_in_range;
+            int value = hits_ratio * 10000000 + values_in_range;
```

Comment:
This will overflow 10M * 10M > 32bit

Reply:
Max value of `hits_ratio` is 200  and max value of `values_in_range` is 1.000.000. 200 * 10.000.000 + 1.000.000 = 2.001.000.000 (will not overflow).